include/pybind11/pybind11.h

```diff
@@ -1381,6 +1382,13 @@ template <typename Type> class enum_ : public class_<Type> {
             }
             return pybind11::str("{}.???").format(name);
         });
+        def_property_readonly("name", [m_entries_ptr](Type value) -> pybind11::str {
+            for (const auto &kv : reinterpret_borrow<dict>(m_entries_ptr)) {
+                if (pybind11::cast<Type>(kv.second[int_(0)]) == value)
+                    return pybind11::str("{}").format(kv.first);
```

Comment:
Going through `format` here is unnecessary: this can just be: `return pybind11::str(kv.first);`